fdbclient/include/fdbclient/TenantEntryCache.actor.h

```diff
@@ -220,7 +222,7 @@ class TenantEntryCache : public ReferenceCounted<TenantEntryCache<T>>, NonCopyab
 			if (!cache->lastTenantId.present()) {
 				return false;
 			}
-			return cache->lastTenantId.get() > 0;
+			return cache->lastTenantId.get() >= 0;
```

Comment:
Why `== 0` is a valid case here? 
> 0 is a valid tenantId (< 0 are reserved) and `0` is an INVALID_TENANT_ID

Reply:
According to AJ `0` is a valid tenant id

Reply:
Interesting! Good to learn that. Thanks